definitions/ext-waf/summary_metrics.yml

```diff
@@ -0,0 +1,26 @@
+cpuUtilization:
```

Comment:
hi @Mesverrum, I think you are missing the goldenMetric field we introduced recently. 

It is part of the longer term goal of moving everything to golden metrics, please take a look here: https://github.com/newrelic/entity-definitions/blob/02e4e618d14f818a44e7fc4ef8cf32e3ff6773d3/docs/summary_metrics.md#defining-summary-metrics

Reply:
@otaviocarvalho i see the examples, updated